services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/ConflictMonitorApiProperties.java

```diff
@@ -0,0 +1,443 @@
+    private String dockerHostIP = null;
+    private static final String DEFAULT_CONNECT_PORT = "8083";
+
+    public int getMaximumResponseSize() {
+        return maximumResponseSize;
+    }
+
+    @Value("${maximumResponseSize}")
+    public void setMaximumResponseSize(int maximumResponseSize) {
+        ConflictMonitorApiProperties.maximumResponseSize = maximumResponseSize;
+    }
+
+    public String getCors() {
+        return cors;
+    }
+
+    @Value("${cors}")
+    public void setCors(String cors) {
+        ConflictMonitorApiProperties.cors = cors;
+    }
+
+    public String getCmServerURL() {
+        return cmServerURL;
+    }
+
+    @Value("${cmServerURL}")
+    public void setCmServerURL(String cmServerUrl) {
+        this.cmServerURL = cmServerUrl;
+    }
+
+    public String getEmailBroker() {
+        return emailBroker;
+    }
+
+    @Value("${emailBroker}")
+    public void setEmailBroker(String emailBroker) {
+        this.emailBroker = emailBroker;
+    }
+
+    public String getEmailFromAddress() {
+        return emailFromAddress;
+    }
+
+    @Value("${emailFromAddress}")
+    public void setEmailFromAddress(String emailFromAddress) {
+        this.emailFromAddress = emailFromAddress;
+    }
+
+    public long getMongoTimeoutMs() {
+        return mongoTimeoutMs;
+    }
+
+    @Value("${mongoTimeoutMs}")
+    public void setMongoTimeoutMs(long mongoTimeoutMs) {
+        this.mongoTimeoutMs = mongoTimeoutMs;
+    }
+
+    public String getKafkaBrokers() {
+        return kafkaBrokers;
+    }
+
+    @Value("${spring.kafka.bootstrap-servers}")
+    public void setKafkaBrokers(String kafkaBrokers) {
+        this.kafkaBrokers = kafkaBrokers;
+    }
+
+    @Value("${kafka.linger_ms}")
+    public void setKafkaLingerMs(int lingerMs) {
+        this.lingerMs = lingerMs;
+    }
+
+    public int getKafkaLingerMs() {
+        return lingerMs;
+    }
+
+    @Value("${enable.api}")
+    public void setEnableApi(boolean enableApi) {
+        this.enableAPI = enableApi;
+    }
+
+    public boolean isApiEnabled() {
+        return enableAPI;
+    }
+
+    @Value("${enable.email}")
+    public void setEnableEmail(boolean enableEmail) {
+        this.enableAPI = enableEmail;
+    }
+
+    public boolean isEmailEnabled() {
+        return enableEmails;
+    }
+
+    @Value("${enable.report}")
+    public void setEnableReports(boolean enableReports) {
+        this.enableReports = enableReports;
+    }
+
+    public boolean isReportsEnabled() {
+        return enableReports;
+    }
+
+    public String getKafkaProducerType() {
+        return kafkaProducerType;
+    }
+
+    public void setKafkaProducerType(String kafkaProducerType) {
+        this.kafkaProducerType = kafkaProducerType;
+    }
+
+    public Boolean getConfluentCloudStatus() {
+        return confluentCloudEnabled;
+    }
+
+    public void setConfluentCloudStatus(boolean confluentCloudStatus) {
+        this.confluentCloudEnabled = confluentCloudStatus;
+    }
```

Comment:
chore(blocking): either use a lombok getter/setter annotation or correct the accessor methods like the following. The getters/setters should return the same data type as the property they're interacting with & they should be named the same.
```suggestion
    public boolean getConfluentCloudEnabled() {
        return confluentCloudEnabled;
    }

    public void setConfluentCloudEnabled(boolean confluentCloudStatus) {
        this.confluentCloudEnabled = confluentCloudStatus;
    }
```